pkg/quota/v1/evaluator/core/pods.go

```diff
@@ -351,6 +351,8 @@ func PodUsageFunc(obj runtime.Object, clock clock.Clock) (corev1.ResourceList, e
 		limits = quota.Max(limits, pod.Spec.InitContainers[i].Resources.Limits)
 	}
 
+	requests = quota.Add(requests, pod.Spec.Overhead)
+	limits = quota.Add(limits, pod.Spec.Overhead)
```

Comment:
I see that there is a feature flag for this:

https://github.com/kubernetes/kubernetes/blob/267afe2358db44dd764b785744448e403359e5fe/pkg/features/kube_features.go#L738

Hence, we should probably wrap this logic with the feature flag.

Reply:
Thanks. Done.